src/Compilers/Core/Portable/Symbols/Attributes/CommonAttributeData.cs

```diff
@@ -255,11 +255,41 @@ internal ObsoleteAttributeData DecodeObsoleteAttribute(ObsoleteAttributeKind kin
                     return DecodeObsoleteAttribute();
                 case ObsoleteAttributeKind.Deprecated:
                     return DecodeDeprecatedAttribute();
+                case ObsoleteAttributeKind.WindowsExperimental:
+                    return DecodeWindowsExperimentalAttribute();
                 case ObsoleteAttributeKind.Experimental:
-                    return DecodeExperimentalAttribute();
+                    return decodeExperimentalAttribute();
                 default:
                     throw ExceptionUtilities.UnexpectedValue(kind);
             }
+
+            ObsoleteAttributeData decodeExperimentalAttribute()
```

Comment:
>decodeExperimentalAttribute [](http://example.com/codeflow?start=34&length=27)

Consider using a regular method, rather than a local function, for consistency with the other cases in the switch above.